modules/mobile/docs/schemas/User.yml

```diff
@@ -62,6 +62,7 @@ properties:
               - DSL
               - MHV
       authorizedServices:
+        description: Services that users are allowed to view.
```

Comment:
Could we change this to something like `Services the user is allowed to access. See availableServices for a list of all services available via the mobile API.`?